### PR TITLE
Exceptions doc strings

### PR DESCRIPTION
**Pull request by schaefi**
Add doc information for Exceptions

### CODE REVIEW

File: test/unit/package_manager_apt_test.py [hunk 1]
```diff
@@ -7,7 +7,9 @@
 from .test_helper import *
 
 from kiwi.package_manager.apt import PackageManagerApt
-from kiwi.exceptions import *
+from kiwi.exceptions import (
+    KiwiDebootstrapError
+)
```
Comment (isbm) [inline test/unit/package_manager_apt_test.py]: No need of parenthesis in this case. Only when multiple imports.

**Comment by isbm** [comment]
Is this intentional having no "." at the end of every sentence? Normally, this documentation you see in IDEs, it is generated etc. So typically it supposed to be just a proper grammar.

**Comment by schaefi** [comment]
No this is just me being lazy, I'll fix that... duh you might see this at several other doc strings *urks*

**Comment by isbm** [comment]
Done. :+1:

**Pull request merged**